vindemitor/core/post_processor.py

```diff
@@ -128,3 +125,19 @@ def _mux_media(self, title: Title_T, tracks: Tracks, progress_callback) -> Path:
             video_track.delete()
 
         return muxed_path
+
+    def _tag_file(self, path, tag, title: Title_T):
+        xml_lines = ['<?xml version="1.0" encoding="UTF-8"?>', "<Tags>", "  <Tag>", "    <Targets/>"]
+        xml_lines.append(f"    <Simple><Name>Description</Name><String>{title.description}</String></Simple>")
+        xml_lines.append(f"    <Simple><Name>Group</Name><String>{tag}</String></Simple>")
+        xml_lines.extend(["  </Tag>", "</Tags>"])
+        with tempfile.NamedTemporaryFile("w", suffix=".xml", delete=False) as f:
+            f.write("\n".join(xml_lines))
+            tmp_path = Path(f.name)
+        subprocess.run(
+            ['mkvpropedit', path, '--tags', f'global:{tmp_path}'],
```

Comment:
See `vindemitor\core\binaries.py`.
This code should not crash for any reason (e.g. missing binary). Execution must continue.